packages/idos-sdk-js/src/lib/enclave-providers/types.ts

```diff
@@ -21,4 +21,9 @@ export interface EnclaveProvider {
   confirm(message: string): Promise<boolean>;
   encrypt(message: Uint8Array, receiverPublicKey?: Uint8Array): Promise<Uint8Array>;
   decrypt(message: Uint8Array, senderPublicKey?: Uint8Array): Promise<Uint8Array>;
+
+  filterCredentialsByCountries?(
```

Comment:
I'd make this mandatory, and have the metamask enclave just raise for now.
```suggestion
  filterCredentialsByCountries(
```